### PR TITLE
[FIX] Remove FeedItem when Event is removed [branch ch678]

### PR DESCRIPTION
**Pull request by PaskR**

### CODE REVIEW

File: src/Repository/CommitteeFeedItemRepository.php [hunk 1]
```diff
@@ -82,4 +83,16 @@ private function createCommitteeTimelineQueryBuilder(string $committeeUuid): Que
 
         return $qb;
     }
+
+    public function removeEventItems(BaseEvent $event)
+    {
+        $qb = $this->createQueryBuilder('i');
+        $qb->delete()
```
Comment (salahm) [inline src/Repository/CommitteeFeedItemRepository.php]: do you really need `expr` here ?

```php
->where('i.itemType = :type')
->andWhere('i.event = :event')
```

isn't enough ?

Comment (PaskR) [reply src/Repository/CommitteeFeedItemRepository.php]: Soz, it's an old habit ;)

**Comment by salahm** [comment]
Also, can't you use doctrine cascade deleting instead of doing it manually ? 
something like "onDelete="CASCADE"  here https://github.com/EnMarche/en-marche.fr/blob/master/src/Entity/CommitteeFeedItem.php#L44 ?

**Comment by PaskR** [comment]
Oh i totally missed this thing ! Thanks

**Comment by PaskR** [comment]
PR redone here : https://github.com/EnMarche/en-marche.fr/pull/1471

**Pull request closed without merging**